inference-engine/src/mkldnn_plugin/nodes/mkldnn_fake_quantize_node.h

```diff
@@ -121,11 +121,22 @@ class MKLDNNFakeQuantizeNode : public MKLDNNNode {
     InferenceEngine::Precision getInputPrecision() const { return inputPrecision; }
     InferenceEngine::Precision getOutputPrecision() const { return outputPrecision; }
 
-    void appendPostOps(mkldnn::post_ops& ops, const VectorDims &postOpDims = {}, int align = -1, bool initAsBinary = false,
-                       bool initBinaryMemory = false) override;
+    // MKLDNN quantization_injectors assumes that quantization data memory is always aligned on 16
+    // by length of AVX512 vector register which is also enough for AVX2 and SSE42 implementations.
+    // Otherwise it can lead to buffer over-read and performance penalties due to denormals.
+    void appendPostOps(mkldnn::post_ops& ops, const VectorDims &postOpDims = {}, int align = 16) override;
```

Comment:
I would say that changing default parameters list in the interface implementation (though internal at the moment) is not good practice since the method being called to the base type and to the specific type without explicit parameters specialization will have different results, which will definitely cause various errors in the code. Could you please align the default parameters list with the interface?

Reply:
Discussed.
Align arguments will be completely removed in scope of follow-up PR